### PR TITLE
Wyatt Benson - Assignment 2

### PR DESCRIPTION
**Pull request by wyatt-benson**

### CODE REVIEW

File: problems/euler/README.md [hunk 1]
```diff
@@ -1 +1,2 @@
 # Euler problems
+Wyatt Benson - Problem 71
```
Comment (sleddog) [inline problems/euler/README.md:2]: I like this idea.  Lets discuss tomorrow in class.

File: problems/euler/problem71/CPlusPlus/WyattBenson/prob71.cpp [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+	int denom;
+	float val;
+
+	struct node*  next;
+
+	void setVal(){
+		val = (float)(num)/(float)(denom);
+	}
+};
+void insert(struct node** head, int n, int d){
```
Comment (DanielBachler) [inline problems/euler/problem71/CPlusPlus/WyattBenson/prob71.cpp:17]: Maybe add a method header comment here explaining what the method does in some level of detail

File: problems/euler/problem71/CPlusPlus/WyattBenson/prob71.cpp [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+				iter->next = temp;;
+			}
+		}
+	}
+	else{
+		temp->next = (*head);
+		(*head) = temp;
+	}
+}
+void solve(struct node* iter){
```
Comment (DanielBachler) [inline problems/euler/problem71/CPlusPlus/WyattBenson/prob71.cpp:41]: Same as above, header comment for method

File: problems/euler/problem71/CPlusPlus/WyattBenson/prob71.cpp [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+	float num = (float)(3)/(float)(7);
+	while((iter->next->val != num) && (iter->next != NULL)){iter = iter->next;}
+	cout << "Answer is " << iter->num << "/" << iter->denom << "\n";
+}
+void generate(struct node** head, int mult,float ans){
```
Comment (DanielBachler) [inline problems/euler/problem71/CPlusPlus/WyattBenson/prob71.cpp:46]: Header comment again

File: problems/euler/problem71/CPlusPlus/WyattBenson/README.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+g++ orgin_file -o output_file
+
+Example: g++ prob71.cpp -o prob71
+
+To run enter file name followed by two arguments
+
+./file_name n a/b
+
+Example: ./prob71 500 3/7
```
Comment (DanielBachler) [inline problems/euler/problem71/CPlusPlus/WyattBenson/README.md:14]: Good README, explains what program does and how to run in detail.

**Review by DanielBachler — APPROVED** [review]
Overall code looks good, probably should add a few comments to the program but the code itself is clean and legible enough that the overall jist of the program was easy enough to understand.  

Good job!
+1

**Pull request merged**